pkg/didcomm/protocol/didexchange/service_test.go

```diff
@@ -627,14 +627,16 @@ func startConsumer(t *testing.T, svc *Service, done chan bool) {
 		}
 	}()
 
-	statusCh := make(chan dispatcher.DIDCommMsg, 10)
+	statusCh := make(chan dispatcher.MsgEvent, 10)
 	err = svc.RegisterMsgEvent(statusCh)
 	require.NoError(t, err)
 	go func() {
 		for e := range statusCh {
-			// receive the events
-			if e.Type == ConnectionRequest {
-				writeToDB(t, svc, e)
+			if e.Type == dispatcher.PreState {
```

Comment:
Does existing test cases cover Post state message events ? I don't see new test cases.

Reply:
I didn't write them yet I just want to make sure the design I followed for post event is good